build.gradle

```diff
@@ -404,7 +404,11 @@ project(':clients') {
       def headRef = file("../.git/HEAD").text
       if (headRef.contains('ref: ')) {
         headRef = headRef.replaceAll('ref: ', '').trim()
-        commitId = file("../.git/$headRef").text.trim().take(takeFromHash)
+        if (file("../.git/$headRef").exists()) {
+          commitId = file("../.git/$headRef").text.trim().take(takeFromHash)
+        } else {
+          commitId
```

Comment:
Are we supposed to assign a value to `commitId` in the `else or am I missing something?